app/mailers/user_mailer.rb

```diff
@@ -29,7 +29,7 @@ def confirmation_instructions(user, token, _options = {})
       rails_mailer: mailer_name,
       rails_mail_template: action_name,
       personalisation: {
-        first_name: user.first_name.capitalize,
+        full_name: user.full_name.capitalize,
```

Comment:
I think we should be consistent with how we show names in our emails. Sign in one uses `titleize` instead of `capitalize` - probably causes some dissonance.

Also, I wonder if we can just use `user.full_name` in both - if the user enters lower case name and don't like it, I imagine we might let them change it? (later down the line)